cupy/core/core.pyx

```diff
@@ -1482,6 +1482,19 @@ cdef class ndarray:
         if update_f_contiguity:
             self._update_f_contiguity()
 
+    cdef ndarray _view(self, const vector.vector[Py_ssize_t]& shape,
+                       const vector.vector[Py_ssize_t]& strides,
+                       bint update_c_contiguity,
+                       bint update_f_contiguity):
+        cdef ndarray v
+        v = ndarray.__new__(ndarray)
+        v.data = self.data
+        v.base = self.base if self.base is not None else self
+        v.dtype = self.dtype
+        v._set_shape_and_strides(
+            shape, strides, update_c_contiguity, update_f_contiguity)
```

Comment:
As neither the contiguity flags are explicitly set, I think `update_c_contiguity` and `update_f_contiguity` should be both `True`.

Reply:
Thanks. I added setting `self`'s flags.